dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -502,7 +501,8 @@ public Host findSystemHost (User user, boolean respectFrontendRoles) throws DotD
                 createSystemHost();
             } else {
                 final String systemHostId = (String) rs.get(0).get("id");
-                this.systemHost = DBSearch(systemHostId, user, respectFrontendRoles);
+                this.systemHost = (Host) APILocator.getContentletAPI().find(systemHostId,user,respectFrontendRoles);
+                //this.systemHost = DBSearch(systemHostId, user, respectFrontendRoles);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)